bokehjs/src/less/main.less

```diff
@@ -77,20 +77,32 @@
   display: block;
 }
 
-.bk-tooltip.bk-left::before {
+.bk-tooltip.bk-left.bk-arrow::before {
```

Comment:
`bk-tooltip-arrow` would be a better name to avoid collisions with arrows in other contexts.